### ISSUE TITLE
Which server languages does Swagger-codegen support?

### ISSUE THREAD

**Issue by WANGMUXIAN** [swagger-api/swagger-codegen#10970]
##### Description

1、https://github.com/swagger-api/swagger-codegen and https://github.com/swagger-api/swagger-codegen/wiki/Server-stub-generator-HOWTO say server support languages not same，which right？
<img width="842" alt="111" src="https://user-images.githubusercontent.com/13382013/114279977-c60e4680-9a69-11eb-87b8-95ffaea0f1cd.png">

<img width="979" alt="222" src="https://user-images.githubusercontent.com/13382013/114279985-cf97ae80-9a69-11eb-8389-a1ef4ffd0808.png">

2、https://editor.swagger.io/,  this page show generate server languages is not always the same， why?

<img width="1213" alt="333" src="https://user-images.githubusercontent.com/13382013/114279991-d9211680-9a69-11eb-9a1b-7f42917c4bfa.png">

<img width="1145" alt="444" src="https://user-images.githubusercontent.com/13382013/114279994-dfaf8e00-9a69-11eb-9a8e-0847b8ede481.png">

3、I use "swagger-codegen generate -i https://petstore.swagger.io/v2/swagger.json -l pistache-server -o ." and get a error,  but I sometime can download pistache-server code in https://editor.swagger.io/, why?

<img width="571" alt="555" src="https://user-images.githubusercontent.com/13382013/114280037-0bcb0f00-9a6a-11eb-844b-5a3a90cb9e3d.png">

##### Swagger-codegen version

3.0.25

**Comment by MagnusNilsson11** [comment]
I agree, very confusing. I also love how you need to create exceptions to get a list of the languages. Should be available from the help menu one would think.
